### PR TITLE
[12.0][MIG] account_invoice_reimbursable

### PR DESCRIPTION
**Pull request by Saran440**
- [x] change function compute to _convert
- [x] Fix bug error when choose currency is not company currency

### CODE REVIEW

File: account_invoice_reimbursable/views/report_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                 <tr class="border-black" >
+                     <td><strong>Executable</strong></td>
+                     <td class="text-right">
+                        <span t-field="o.executable_total" t-options='{"widget": "monetary", "display_currency": o.currency_id}'/>
+                    </td>
+                 </tr>
+             </t>
+         </xpath>
+    </template>
+</odoo>
```
Comment (tbaden) [inline account_invoice_reimbursable/views/report_invoice.xml]: please add missing newline

File: account_invoice_reimbursable/views/account_invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            </notebook>
+            <xpath expr="//group[hasclass('oe_subtotal_footer')]/field[@name='amount_total']" position="after">
+                <field name="reimbursable_count" invisible="1"/>
+                <field name="reimbursable_total" attrs="{'invisible': [('reimbursable_count', '=', 0)]}"/>
+                <field name="executable_total" attrs="{'invisible': [('reimbursable_count', '=', 0)]}" class="oe_subtotal_footer_separator"/>
+            </xpath>
+        </field>
+    </record>
+
+</odoo>
```
Comment (tbaden) [inline account_invoice_reimbursable/views/account_invoice_view.xml]: please add missing newline

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review: LGTM just something minor

**Comment by Saran440** [comment]
Fixed it. 
@tbaden Thank you for your reviewed.

**Review by tbaden — APPROVED** [review]

**Review by jarroyomorales — APPROVED** [review]

**Comment by jarroyomorales** [comment]
@pedrobaeza Can this be merged?

**Comment by pedrobaeza** [comment]
OK, merging blindly due to 2 reviews.

/ocabot merge

**Pull request closed without merging**